### PR TITLE
feat(SoundType): add two differents types of sound

### PR DESCRIPTION
**Pull request by Steci**
How to handle game sound

### CODE REVIEW

File: include/common/Elements.hpp [hunk 1]
```diff
@@ -21,7 +21,8 @@ namespace arcade::common {
     };
 
     enum class SoundType {
-
+        LOOP,
+        EVENT,
```
Comment (Steci) [inline include/common/Elements.hpp:25]: Should be tempory in game data (remove in next tick)

File: include/common/Elements.hpp [hunk 2]
```diff
@@ -21,7 +21,8 @@ namespace arcade::common {
     };
 
     enum class SoundType {
-
+        LOOP,
```
Comment (Steci) [inline include/common/Elements.hpp:24]: for background music

File: include/common/Elements.hpp (hunk 1 above)
Comment (RezaRahemtola) [reply include/common/Elements.hpp:25]: hmmmm that will be a very very fast sound then

Comment (Tomi-Tom) [reply include/common/Elements.hpp:25]: No need to remove in next tick if we want to use it again, it would be more optimized to store them

**Review by Tomi-Tom — APPROVED** [review]
Noice noice, Gutt gutt

**Comment by Croos3r** [comment]
Here he comes

**Pull request closed without merging**